include/albatross/src/evaluation/model_metrics.hpp

```diff
@@ -64,6 +65,27 @@ struct LeaveOneOutLikelihood
   }
 };
 
+template <typename PredictType, typename FeatureType>
```

Comment:
did you want to do the thing where you go:
```
template <typename FeatureType, typename PredictType = JointDistribution>
```
?

Reply:
I had some templating errors when I tried the first time... but I think I just missed one of the three spots that are affected. I'll update with my next try to get clang formatting correct.